explainer.md

```diff
@@ -0,0 +1,369 @@
+# Secure Payment Confirmation explained
+
+## tl;dr
+
+**Secure Payment Confirmation** is a proposed Web API that enables the use of
+biometric authentication in payment flows on the web, building on top of
+[WebAuthn]. The goal is to provide strong, low-friction authentication of a
+customer in a payment context, whilst still preserving user privacy better than
+current fingerprinting-based methods.
+
```

Comment:
What would you think of using the same abstract from the spec as the explainer intro?

Reply:
Hrm, I agree in the general goal of aligning them, but in reviewing the spec one:

> Secure Payment Confirmation (SPC) is a Web API to support streamlined authentication during a payment transaction. It is designed to scale authentication across merchants, to be used within a wide range of authentication protocols, and to produce cryptographic evidence that the user has confirmed transaction details.

I feel like I also want WebAuthn mentioned explicitly in the explainer, as well as user privacy.

Reply:
How about appending to the abstract a new paragraph:

"SPC adds payment-specific capabilities atop [WebAuthn] and is designed with stronger privacy protections than risk analysis approaches that rely on data collection."

Reply:
Sounds good, done. I'll send a separate PR to the abstract later.